README.md

```diff
@@ -48,11 +48,17 @@ of specific statistics:
 An additional configuration option is available:
 
 * HTTPTimeout - Integer value in seconds before timing out when connecting
-                to the RabbitMQ Management API. Defaults to 60 seconds.
+to the RabbitMQ Management API. Defaults to 60 seconds.
 * FieldLength - Set the number of characters used to encode dimension data.
-                This option should only ever be set if you specifically
-                compiled collectd with a non-default value for
-                DATA_MAX_NAME_LEN in plugin.h.
+This option should only ever be set if you specifically compiled collectd with
+a non-default value for DATA_MAX_NAME_LEN in plugin.h.
+* VerbosityLevel - Representation of the quantity of metrics generated by
+RabbitMQ that are actually reported to SignalFx. If not specified, the default
+value is level 1. The verbosity levels are as follows:
+
+        1: Only the most commonly-used metrics are reported
+        2: Additonal metrics useful for debugging are reported
+        3: All available metrics are reported
```

Comment:
I would prefer words instead of numbers for the verbosity.  Perhaps  Verbosity: INFO DEBUG TRACE.